docs/app/documentation/component-docs/localization-editor/localization-editor-header/localization-editor-header.component.html

```diff
@@ -0,0 +1,10 @@
+<header>Localization Editor</header>
+<description>
+    Localization Editor is used to display a bunch of fields like inputs or text areas inside popover component. To every
+    single field there is added input-group's add-on.  There are also popover configuration inputs allowed, to define
+    position, or trigger event.
```

Comment:
The localization editor is used to translate text into a number of languages. It promotes spatial efficiency through the use of the popover component under the hood.

maybe this? It shouldn't be a very technical description as designers might read this too